### PR TITLE
field enlarged and db upgrade

### PR DESCRIPTION
**Pull request by DaanHoogland**
### Description

This PR...

Fixes: #8669

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)
- [ ] build/CI

#### Feature/Enhancement Scale

- [ ] Major
- [ ] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [ ] Minor
- [ ] Trivial

### CODE REVIEW

File: engine/schema/src/main/resources/META-INF/db/schema-41900-41910.sql [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+-- "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+-- KIND, either express or implied.  See the License for the
+-- specific language governing permissions and limitations
+-- under the License.
+
+--;
+-- Schema upgrade from 4.19.0.0 to 4.19.1.0
+--;
+
+ALTER TABLE cloud.vm_template MODIFY COLUMN url VARCHAR(1024) DEFAULT NULL COMMENT 'the url where the template exists externally';
```
Comment (weizhouapache) [inline engine/schema/src/main/resources/META-INF/db/schema-41900-41910.sql:22]:
```suggestion
ALTER TABLE `cloud`.`vm_template` MODIFY COLUMN `url` VARCHAR(1024) DEFAULT NULL COMMENT 'the url where the template exists externally';
```

**Comment by weizhouapache** [comment]
@DaanHoogland 
there are several `UnusedImports`

File: engine/schema/src/main/resources/META-INF/db/schema-41900-41910.sql [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+-- "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+-- KIND, either express or implied.  See the License for the
+-- specific language governing permissions and limitations
+-- under the License.
+
+--;
+-- Schema upgrade from 4.19.0.0 to 4.19.1.0
+--;
+
+ALTER TABLE `cloud`.`vm_template` MODIFY COLUMN `url` VARCHAR(1024) DEFAULT NULL COMMENT 'the url where the template exists externally';
```
Comment (weizhouapache) [inline engine/schema/src/main/resources/META-INF/db/schema-41900-41910.sql:22]: EOF need to be fixed

other than it, all else looks good to me

**Review by weizhouapache — APPROVED** [review]

**Comment by DaanHoogland** [comment]
@blueorangutan package

**Comment by DaanHoogland** [comment]
@blueorangutan test alma9 vmware-70u3

**Comment by DaanHoogland** [comment]
@blueorangutan package

**Comment by DaanHoogland** [comment]
@blueorangutan package

**Comment by DaanHoogland** [comment]
@blueorangutan test alma9 kvm-alma9

**Comment by weizhouapache** [comment]
@DaanHoogland 

```
        final String scriptFile = "META-INF/db/schema-41900to41910.sql";
```

**Comment by DaanHoogland** [comment]
> @DaanHoogland
> 
> ```
>         final String scriptFile = "META-INF/db/schema-41900to41910.sql";
> ```

yes?

**Comment by weizhouapache** [comment]
> > @DaanHoogland
> > 
> > ```
> >         final String scriptFile = "META-INF/db/schema-41900to41910.sql";
> > ```
> 
> yes?

The new file in this PR is
engine/schema/src/main/resources/META-INF/db/schema-41900-41910.sql

Not 41900 "to" 41910

@DaanHoogland

**Comment by DaanHoogland** [comment]
@blueorangutan package

**Comment by DaanHoogland** [comment]
@blueorangutan test alma9 kvm-alma9

**Review by weizhouapache — APPROVED** [review]
code lgm

@DaanHoogland 
lint error

File: engine/schema/src/main/resources/META-INF/db/schema-41900to41910.sql [hunk 3]
```diff
@@ -44,4 +48,4 @@ CREATE TABLE IF NOT EXISTS `cloud_usage`.`usage_vpc` (
   PRIMARY KEY (`id`)
 ) ENGINE=InnoDB CHARSET=utf8;
 
-CALL `cloud_usage`.`IDEMPOTENT_ADD_COLUMN`('cloud_usage.cloud_usage', 'state', 'VARCHAR(100) DEFAULT NULL');
+CALL `cloud_usage`.`IDEMPOTENT_ADD_COLUMN`('cloud_usage.cloud_usage', 'state', 'VARCHAR(100) DEFAULT NULL');
```
Comment (DaanHoogland) [inline engine/schema/src/main/resources/META-INF/db/schema-41900to41910.sql:51]:
```suggestion
CALL `cloud_usage`.`IDEMPOTENT_ADD_COLUMN`('cloud_usage.cloud_usage', 'state', 'VARCHAR(100) DEFAULT NULL');
```

**Comment by DaanHoogland** [comment]
@blueorangutan package

**Review by sureshanaparti — APPROVED** [review]
Verified column update query, LGTM

**Comment by sureshanaparti** [comment]
Merging this based on LGTMs / tests.

**Pull request merged**